worlds/sc2wol/Locations.py

```diff
@@ -314,9 +258,15 @@ def get_locations(world: Optional[MultiWorld], player: Optional[int]) -> Tuple[L
                      lambda state: state._sc2wol_has_competent_comp(world, player)),
         LocationData("Shatter the Sky", "Shatter the Sky: Leviathan", SC2WOL_LOC_ID_OFFSET + 2805,
                      lambda state: state._sc2wol_has_competent_comp(world, player)),
-        LocationData("Shatter the Sky", "Beat Shatter the Sky", None,
-                     lambda state: state._sc2wol_has_competent_comp(world, player)),
         LocationData("All-In", "All-In: Victory", None)
     ]
 
-    return tuple(location_table)
+    beat_events = []
+
+    for location_data in location_table:
+        if location_data.name.endswith((": Victory", ": Defeat")):
+            beat_events.append(
+                location_data._replace(name="Beat " + location_data.name.rsplit(": ", 1)[0], code=None)
```

Comment:
A function o generate the name?

Reply:
if there's at least two places where this code would get used, sure.

Reply:
Sometimes I do that even if isn't when it provides a better readibility